scylla-cdc-replicator/src/replication_tests.rs

```diff
@@ -1296,4 +1296,49 @@ mod tests {
             .await
             .unwrap();
     }
+
+    #[tokio::test]
+    async fn test_range_delete() {
+        let schema = TestTableSchema {
+            name: "RANGE_DELETE".to_string(),
+            partition_key: vec![("pk1", "int"), ("pk2", "int")],
+            clustering_key: vec![("ck1", "int"), ("ck2", "int"), ("ck3", "int")],
+            other_columns: vec![("v", "int")],
+        };
+
+        let operations = vec![
```

Comment:
I have a feeling that with such a small number of rows inserted we risk the tests not being accurate. I'd recommend inserting more rows - for example, insert a row for each combination of ck1, ck2, ck3 being in range `1..4` (64 inserts, can be automatically generated and inserted into `operations` in the test).